apps/dev/astro/auth.config.ts

```diff
@@ -0,0 +1,12 @@
+import { defineConfig } from '@auth/astro/config'
+import type { Provider } from '@auth/core/providers'
+import GitHub from '@auth/core/providers/github'
+
+export default defineConfig({
+  providers: [
+    GitHub({
+      clientId: import.meta.env.GITHUB_CLIENT_ID,
+      clientSecret: import.meta.env.GITHUB_CLIENT_SECRET,
+    }) as Provider,
```

Comment:
Is there a reason for this provider needing to have a type assertion?

Reply:
Oops. I think this was left in from a type mismatch issue I was working on. It looks like it works now though, so these can go

Reply:
Fixed